### PR TITLE
PMA-057(Modal window translation)

### PR DESCRIPTION
**Pull request by allyksander**
1. Add translation of modal fields

### CODE REVIEW

File: src/constants/formfields.ts [hunk 1]
```diff
@@ -72,168 +72,171 @@ export const passwordAuthInput: FormField = {
...
+    {
+      registerOptions: {
+        required: 'BoardModalText.REQUIRED',
+        maxLength: {
+          value: 20,
+          message: 'nameErrors.maxLength',
+        },
+        minLength: {
+          value: 1,
+          // TODO ADD translation for '1' value
```
Comment (AlexanderSUS) [inline src/constants/formfields.ts]: Why you miss TODO ? Current min length for  min='2' case

File: src/constants/formfields.ts [hunk 2]
```diff
@@ -72,168 +72,171 @@ export const passwordAuthInput: FormField = {
...
       },
-      minLength: {
-        value: 1,
-        // TODO ADD translation for '1' value
-        message: 'nameErrors.minLength',
+      name: 'title',
+      type: 'text',
+      label: 'BoardModalText.TITLE_LABEL',
+      placeholder: 'BoardModalText.TITLE_PLACEHOLDER',
+      // TODO add translation
```
Comment (AlexanderSUS) [inline src/constants/formfields.ts]: If you add  localization for this case, why you did't remove comment?

File: src/constants/formfields.ts [hunk 3]
```diff
@@ -72,168 +72,171 @@ export const passwordAuthInput: FormField = {
...
+    {
+      registerOptions: {
+        required: 'ListModalText.REQUIRED',
+        maxLength: {
+          value: 20,
+          message: 'nameErrors.maxLength',
+        },
+        minLength: {
+          value: 1,
+          // TODO ADD translation for '1' value
```
Comment (AlexanderSUS) [inline src/constants/formfields.ts]: The same as above

File: src/constants/formfields.ts [hunk 4]
```diff
@@ -72,168 +72,171 @@ export const passwordAuthInput: FormField = {
...
+          message: 'nameErrors.minLength',
+        },
       },
+      name: 'title',
+      type: 'text',
+      label: 'ListModalText.TITLE_LABEL',
+      placeholder: 'ListModalText.TITLE_PLACEHOLDER',
+      // TODO add translation
```
Comment (AlexanderSUS) [inline src/constants/formfields.ts]: Remove this comment if it not actual any more. TODO not only for me, but for all team members

File: src/constants/formfields.ts [hunk 5]
```diff
@@ -72,168 +72,171 @@ export const passwordAuthInput: FormField = {
...
-    placeholder: 'Input your password for confirmation',
-  },
+  fields: [
+    {
+      ...nameAuthInput,
+      placeholder: 'AuthText.NAME_PLACEHOLDER_EDIT',
+    },
+    {
+      ...passwordAuthInput,
+      // TODO add translation
```
Comment (AlexanderSUS) [inline src/constants/formfields.ts]: The same as above

File: src/constants/formfields.ts [hunk 6]
```diff
@@ -72,168 +72,171 @@ export const passwordAuthInput: FormField = {
...
-    placeholder: 'Input your password for confirmation',
-  },
+  fields: [
+    {
+      ...loginAuthInput,
+      placeholder: 'AuthText.LOGIN_PLACEHOLDER_EDIT',
+    },
+    {
+      ...passwordAuthInput,
+      // TODO add translation
```
Comment (AlexanderSUS) [inline src/constants/formfields.ts]: The same as above

**Review by AlexanderSUS — CHANGES_REQUESTED** [review]
Leave comments, please check TODOs, if done remove comments

File: src/constants/formfields.ts (hunk 1 above)
Comment (allyksander) [reply src/constants/formfields.ts]: Fixed.

File: src/constants/formfields.ts (hunk 2 above)
Comment (allyksander) [reply src/constants/formfields.ts]: Fixed.

File: src/constants/formfields.ts (hunk 3 above)
Comment (allyksander) [reply src/constants/formfields.ts]: Fixed.

File: src/constants/formfields.ts (hunk 4 above)
Comment (allyksander) [reply src/constants/formfields.ts]: Fixed.

File: src/constants/formfields.ts (hunk 5 above)
Comment (allyksander) [reply src/constants/formfields.ts]: Fixed.

File: src/constants/formfields.ts (hunk 6 above)
Comment (allyksander) [reply src/constants/formfields.ts]: Fixed.

**Review by AlexanderSUS — APPROVED** [review]

**Pull request merged**